Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -564,6 +597,11 @@ public class PBXProjGenerator {
             (try path.children().sorted(), path)
 
         let excludedFiles: [String] = [".DS_Store"]
+        // Don't process the files of a bundle. Bundles are a special file type
+        // in Xcode ( not directories ).
+        if path.extension == "bundle" {
```

Comment:
Couldn't all the handling of bundles happen right here, instead of all around the place?

Reply:
I think this would be pretty nice